sass/_valkey.scss

```
@@ -1761,7 +1761,9 @@ html.banner-hidden .banner {
 // Documentation Section Styles
 .documentation-section {
   padding: 4rem 2rem;
-  background: #6983FF;
+  // Continues the .whats-new-section gradient, finishing on the #6983ff that
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)